### ISSUE TITLE
[Bug] Ignored packages shown in CLI when writing a changeset

### ISSUE THREAD

**Issue by quantizor** [changesets/changesets#1325]
## Affected Packages

`@changesets/cli`

## Problem

I have three packages in a monorepo, and only one of them is actually publishable (the others have `private: true`.) When I run `yarn changeset`, it asks me which ones I want to write changesets for, _including_ private packages. I very rarely change these other packages and dislike that the CLI forces me to choose which package to bump every time when the others are never relevant.

## Proposed solution

1. Allow config to focus the CLI, specifically `changeset add` to a specific package or list of packages, or
2. Add a flag to `changeset add` to omit unpublishable packages

**Comment by quantizor** [comment]
Digging into the code, I noticed that there is an explicit check to ensure that `version` is not set for packages marked as `private: true`, why is that? Would it be OK if I removed that limitation?

**Comment by altano** [comment]
Are you referring to this check?: https://github.com/changesets/changesets/blob/0bf89b3709e3e3df6ed5dbb8ece0fb000a55d5f4/packages/cli/src/commands/add/isListablePackage.ts#L11

That's actually reading out of the config. So if you set `"privatePackages": false` in your config.json, it will ignore private packages (whether they have a version or not).

**Comment by bluwy** [comment]
Yeah if you don't want to version the private packages, set `privatePackages.version: false`. In a recent change, the CLI will also now add a `(private)` hint if the package is private, which can be helpful in picking the packages.

I believe this can be resolved this way, so I'll close this
